tabs/failsafe.js

```diff
@@ -225,6 +231,7 @@ TABS.failsafe.initialize = function (callback, scrollPosition) {
         $('input[name="failsafe_off_delay"]').val(FAILSAFE_CONFIG.failsafe_off_delay);
         $('input[name="failsafe_throttle_low_delay"]').val(FAILSAFE_CONFIG.failsafe_throttle_low_delay);
         $('input[name="failsafe_delay"]').val(FAILSAFE_CONFIG.failsafe_delay);
+        $('input[name="failsafe_min_distance"]').val(FAILSAFE_CONFIG.failsafe_min_distance);
```

Comment:
Like I wrote before, older versions of firmeare (1.6) might not sent this data. Firmware version should be checked. I suggest allow for it only starting from gte 1.7.4